lib/Capabilities.php

```diff
@@ -270,6 +272,7 @@ public function getCapabilities(): array {
 			$capabilities['config']['chat']['read-privacy'] = $this->talkConfig->getUserReadPrivacy($user->getUID());
 			$capabilities['config']['chat']['typing-privacy'] = $this->talkConfig->getUserTypingPrivacy($user->getUID());
 			$capabilities['config']['call']['blur-virtual-background'] = $this->talkConfig->getBlurVirtualBackground($user->getUID());
+			$capabilities['config']['conversations']['compact-list'] = $this->talkConfig->getConversationsListStyle($user->getUID());
```

Comment:
You already handled this above, the other methods are just not "nullable" so they can't be moved up:
```suggestion
```